.gitignore

```diff
@@ -14,4 +14,5 @@ Pathogen:
 custom_plan.rb
 zeus.json
 .bundle
-.idea/**
\ No newline at end of file
+.idea/**
+public/**
```

Comment:
Yes, I think these should stay. `public` is mostly auto-generated, and `.idea` is temporary files for your IDE, right?